charts/humio-strix/templates/deployment.yaml

```diff
@@ -98,6 +98,22 @@ spec:
             - name: PERF_USERS
               value: {{ default "3" .Values.simulationOptions.hecSimulation.users | quote }}
             {{- end }}
+            {{- if eq .Values.simulation "HECRandomnessSimulation" }}
+            - name: RANDOMNESS
+              value: {{ default "3" .Values.simulationOptions.hecRandomnessSimulation.randomness | quote }}
+            - name: PERF_BULK_SIZE
+              value: {{ default "100" .Values.simulationOptions.hecRandomnessSimulation.bulkSize | quote }}
```

Comment:
I think we should push this to 200 per our suggestion on filebeat: https://docs.humio.com/integrations/data-shippers/beats/filebeat/